src/Formatter/JmsSerializer.php

```diff
@@ -23,7 +23,7 @@ public function __construct(SerializerInterface $serializer, string $format)
     /**
      * {@inheritdoc}
      */
-    public function format($content, array $attributes = []): string
+    public function formatContent($content, array $attributes = []): string
```

Comment:
This should be protected or have the method as public on the parent class

Reply:
changed to public in parent